kairon/data_processor/processor.py

```diff
@@ -1053,8 +1064,8 @@ def add_training_data(self, training_data: List[models.TrainingData], bot: Text,
             story_name = "path_" + data.intent
             utterance = "utter_" + data.intent
             events = [
-                {"name": data.intent, "type": "user"},
-                {"name": utterance, "type": "action"}]
+                {"name": data.intent, "type": "user", "entities":[]},
+                {"name": utterance, "type": "action", "entities":[]}]
```

Comment:
user action does not have entities field so entities in the db must be null

Reply:
no it has empty list